File: CS/E2876.Web/ReportService1.svc.cs
using System.ServiceModel;
using DevExpress.Data.Utils.ServiceModel;
using DevExpress.XtraReports.UI;
// ...

namespace E2876.Web {
    // NOTE: You can use the "Rename" command on the "Refactor" menu to change the class name "ReportService1" in code, svc and config file together.
    [SilverlightFaultBehavior]
    public class ReportService1 : DevExpress.XtraReports.Service.ReportService {
        protected override void FillDataSources(XtraReport report, string reportName, bool isDesignActive) {
        }

        protected override void SaveReportLayout(string reportName, byte[] layoutData) {
            throw new FaultException("This method is not implemented.");
        }
    }
}

File: VB/E2876.Web/ReportService1.svc.vb
Imports Microsoft.VisualBasic
Imports System.ServiceModel
Imports DevExpress.Data.Utils.ServiceModel
Imports DevExpress.XtraReports.UI
' ...

Namespace E2876.Web
	' NOTE: You can use the "Rename" command on the "Refactor" menu to change the class name "ReportService1" in code, svc and config file together.
	<SilverlightFaultBehavior> _
	Public Class ReportService1
		Inherits DevExpress.XtraReports.Service.ReportService
		Protected Overrides Sub FillDataSources(ByVal report As XtraReport, ByVal reportName As String, ByVal isDesignActive As Boolean)
		End Sub

		Protected Overrides Sub SaveReportLayout(ByVal reportName As String, ByVal layoutData() As Byte)
			Throw New FaultException("This method is not implemented.")
		End Sub
	End Class
End Namespace
